requirements/install.txt

```diff
@@ -0,0 +1,8 @@
+# these are dependencies required to use the package
+# ought to mirror 'instal_requires' under 'options' in config.cfg
+astropy >= 3.1
+colorama >= 0.3
```

Comment:
Just FYI - #728 will remove colorama as a dependency

Reply:
That means I should do a little code review....😁

Reply:
Meanwhile #675 adds numba and xarray ;)